docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -2475,26 +2493,31 @@ Subsequently, the bundles can be used at runtime:
 <3> `Lucie` is the parameter of the message bundle interface method.  
 <4> It is also possible to obtain a localized message for a key resolved at runtime using a reserved key `message`. The validation is skipped in this case though.
 
-.Message Bundle Interface Example
+
+==== Default Bundle Name
+
+The bundle name is defaulted unless it's specified with `@MessageBundle#value()`.
+For a top-level class the `msg` value is used by default.
+For a nested class the name starts with `msg` followed by an undercore, followed by the simple names of all enclosing classes in the hierarchy (top-level class goes first) seperated by underscores.
+
+For example, the name of the following message bundle will be defaulted to `msg_Index`:
+
 [source,java]
 ----
-import io.quarkus.qute.i18n.Message;
-import io.quarkus.qute.i18n.MessageBundle;
+class Index {
 
-@MessageBundle <1>
-public interface AppMessages {
+    @MessageBundle
+    interface Bundle {
```

Comment:
It would be nice to document what's the expected .properties file name in this case

Reply:
Ok, I can move the `NOTE: The bundle name is also u....` from the following section here...